### PR TITLE
started implementing exercise event rendering.

### PR DESCRIPTION
**Pull request by cbouwense**

### CODE REVIEW

**Comment by benwaffle** [comment]
pls merge master

File: client/src/AppRouter.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Route, HashRouter, Switch } from 'react-router-dom';
+import App from './App';
+import { Home, Login, Register } from './components';
+
+const AppRouter = () => {
```
Comment (benwaffle) [inline client/src/AppRouter.jsx]: why delete Router.jsx

File: client/src/components/ExerciseList/ExerciseEvent.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <div className="card">
+      <div className="card-content">
+        <div className="media">
+          <div className="media-content">
+            <p className="title is-4">{props.event.exercise.name}</p>
+          </div>
+        </div>
+
+        <div className="content">
+          {fieldsToRender.map(f => f)}
```
Comment (benwaffle) [inline client/src/components/ExerciseList/ExerciseEvent.jsx]: useless map

File: client/src/components/ExerciseList/ExerciseEventList.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { ExerciseEvent } from './ExerciseEvent';
+
+export const ExerciseEventList = ({ exerciseList }) => (
+  <div>
+    <ul>
+      {exerciseList.map(e => (
+        <li>
```
Comment (benwaffle) [inline client/src/components/ExerciseList/ExerciseEventList.jsx]: add `key={i}`

File: client/src/components/ExerciseList/ExerciseListContainer.jsx [hunk 4]
```diff
@@ -33,3 +33,19 @@ class ExerciseListContainer extends React.Component {
 }
 
 export default ExerciseListContainer;
+
+import { connect } from 'react-redux';
+import ExerciseEventList from './ExerciseEventList';
+
+const mapStateToProps = state => {
+  const user_id = state;
```
Comment (benwaffle) [inline client/src/components/ExerciseList/ExerciseListContainer.jsx]: see the shape of our state in `src/store/initialState.js`

File: client/src/components/ExerciseList/ExerciseListContainer.jsx [hunk 5]
```diff
@@ -33,3 +33,19 @@ class ExerciseListContainer extends React.Component {
...
+
+    // Get the exercise event list from the db
+    const user = axios.get(`/api/${user_id}`);
```
Comment (benwaffle) [inline client/src/components/ExerciseList/ExerciseListContainer.jsx]: you shouldn't make network calls here, in general you should create an action creator in `src/actions`, map it here in mapDispatchToProps, and call it on `componentDidMount`. BTW we have exerciseEvents already in the store

File: client/src/AppRouter.jsx (hunk 1 above)
Comment (TaberIV) [reply client/src/AppRouter.jsx]: Right I should have just asked about that, but it's exporting the component AppRouter so why shouldn't it be named that?

File: client/src/components/ExerciseList/ExerciseEvent.jsx (hunk 2 above)
Comment (TaberIV) [reply client/src/components/ExerciseList/ExerciseEvent.jsx]: I hadn't touched ExerciseEvent.jsx until now but I'll make that change

File: client/src/AppRouter.jsx (hunk 1 above)
Comment (benwaffle) [reply client/src/AppRouter.jsx]: ok

Comment (benwaffle) [reply client/src/AppRouter.jsx]: but when you merge, make sure AppRouter keeps the latest routes

File: client/src/components/ExerciseList/ExerciseEventList.jsx (hunk 3 above)
Comment (TaberIV) [reply client/src/components/ExerciseList/ExerciseEventList.jsx]: Done

File: client/src/components/ExerciseList/ExerciseListContainer.jsx (hunk 4 above)
Comment (TaberIV) [reply client/src/components/ExerciseList/ExerciseListContainer.jsx]: Does user store the user's id or the user as an object?

File: client/src/components/ExerciseList/ExerciseListContainer.jsx (hunk 5 above)
Comment (TaberIV) [reply client/src/components/ExerciseList/ExerciseListContainer.jsx]: Oh this answers my previous question

File: client/src/components/ExerciseList/ExerciseEventList.jsx [hunk 6]
```diff
@@ -1,14 +1,15 @@
...
 
 const ExerciseEventList = ({ exerciseList }) => (
   <div>
     <ul>
-      {exerciseList.map((e, i) => (
-        <li key={i}>
-          <ExerciseEvent exercise={e} />
-        </li>
-      ))}
+      {exerciseList > 0 &&
```
[the remark this answers is not in the document]
Comment (TaberIV) [reply client/src/components/ExerciseList/ExerciseEventList.jsx]: Oops the > 0 is there by mistake

**Pull request merged**